### PR TITLE
Add docs about the Fail2ban role

### PR DESCRIPTION
**Pull request by raph-topo**
Companion for https://github.com/roots/trellis/pull/1177.

### CODE REVIEW

File: docs/trellis/master/fail2ban.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+description: Trellis allows you to customize Fail2ban services and filters.
+---
+
+# Fail2ban
+
+Trellis install Fail2ban on your servers.
```
Comment (swalkinshaw) [inline docs/trellis/master/fail2ban.md:7]: "installs"

**Comment by TangRufus** [comment]
With https://github.com/roots/trellis/pull/1177 merged, do you think this PR is ready for review @raph-topo?

**Comment by raph-topo** [comment]
I believe so. But I'll review it again in a few days to add what I learnt from a few month's usage.

**Comment by swalkinshaw** [comment]
Looks good to me 👍

**Comment by raph-topo** [comment]
Hello.
Sorry that this is not ready yet. I am investigating issues where Fail2ban triggers too eagerly on provisioning.

**Pull request closed without merging**